src/components/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Person } from '../../types/Person';
+
+type Props = {
+  people: Person[],
+  onSelected: (person: Person) => void,
+};
+
+export const DropdownMenu: React.FC<Props> = React.memo(
+  ({ people, onSelected }) => {
+    return (
+      <div
+        className="dropdown-menu"
+        role="menu"
+        id="dropdown-menu"
+      >
+        <div className="dropdown-content">
+          {people.length === 0
```

Comment:
It's better to use just people.length, since u use it with ternary operator as Max stated in Slack.
![image](https://github.com/mate-academy/react_autocomplete/assets/128422319/40f4999a-f16e-492f-bbf1-3479ac34bd8f)